src/core/stripAndCollapse.js

```diff
@@ -0,0 +1,12 @@
+define( function() {
+	"use strict";
+
+	// Strip and collapse whitespace according to HTML spec
+	// https://html.spec.whatwg.org/multipage/infrastructure.html#strip-and-collapse-whitespace
+	var rhtmlSpace = /[\x20\t\r\n\f]+/g,
```

Comment:
It's more characters and a space is a space. :) But I see we're using it in other places like that already. Changing all instances to a space saves only one byte gzipped.